src/psyclone/psyir/transformations/__init__.py

```diff
@@ -99,4 +102,6 @@
            'FoldConditionalReturnExpressionsTrans',
            'HoistTrans',
            'OMPTaskwaitTrans',
-           'LoopTiling2DTrans']
+           'LoopTiling2DTrans',
```

Comment:
I suspect (but haven't discussed with @sergisiso or @rupertford ) that we don't need to list the classes - this is only required if we do `from xxx import *` and we never do that - and it causes errors when building the reference guide.